sql/expression/function/load_file.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package function
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+type LoadFile struct {
+	fileName sql.Expression
+}
+
+var _ sql.FunctionExpression = (*LoadFile)(nil)
+
+// NewLoadFile returns a LoadFile object for the LOAD_FILE() function.
+func NewLoadFile(ctx *sql.Context, fileName sql.Expression) sql.Expression {
+	return &LoadFile{
+		fileName: fileName,
+	}
+}
+
+// Resolved implements sql.Expression
```

Comment:
My other comment is far more important, but while you're here might as well make your comments consistent. Some end with a period and others do not for mostly the same text.